src/Helpers/QueriesRelationships.php

```diff
@@ -128,6 +130,26 @@ public function addHybridHas(Relation $relation, $operator = '>=', $count = 1, $
         return $this->whereIn($this->getRelatedConstraintKey($relation), $relatedIds, $boolean, $not);
     }
 
+    /**
+     * @param Relation $relation
+     *
+     * @return void
+     *
+     * @throws Exception
+     */
+    public function assertHybridRelationSupported(Relation $relation): void
```

Comment:
If this method don't need to be public, make it private so that we don't have to maintain backward compatibility on it.

```suggestion
    private function assertHybridRelationSupported(Relation $relation): void
```

In PHPLIB we enforce private methods to be at after the public ones, I think that would be a good idea for this method to move it at the end.

Reply:
Agree, I made it public cause I thought the other usage was outside of the class (it's not)